crates/shim-protos/examples/ttrpc-client.rs

```diff
@@ -13,11 +13,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use containerd_shim_protos::{api::CreateTaskRequest, TaskClient};
+use containerd_shim_protos::api::CreateTaskRequest;
+
+#[cfg(not(feature = "async"))]
+use containerd_shim_protos::{Client, TaskClient};
+
+#[cfg(feature = "async")]
+use containerd_shim_protos::shim::shim_ttrpc_async::TaskClient;
+#[cfg(feature = "async")]
+use ttrpc::asynchronous::Client;
```

Comment:
Should we reexport `ttrpc::asynchronous::Client` in `containerd_shim_protos::shim::shim_ttrpc_async`, so it gets symmetry to synchronous version.

Reply:
good point, already resolved, thanks